src/trading/trade-until-amount-is-zero.js

```diff
@@ -72,8 +75,12 @@ function tradeUntilAmountIsZero(p) {
           if (p.doNotCreateOrders) return p.onSuccess(tradeOnChainAmountRemaining.toFixed());
           return p.onFailed(new Error("Trade completed but amount of trade unchanged"));
         }
+        var newAmount = convertOnChainAmountToDisplayAmount(tradeOnChainAmountRemaining, tickSize);
```

Comment:
I think we should add `"sharesProvided"` to the `immutableDelete` array in `tradePayload` definition (https://github.com/AugurProject/augur.js/pull/391/files#diff-ca59c39682aec540f1296b9a53f3d384L55), so an argument of this name isn't passed to the underlying API.